### ISSUE TITLE
Copying a devfile's text from the registry viewer adds unneccessary whitespace

### ISSUE THREAD

**Issue by johnmcollier** [devfile/api#596]
/kind bug

### Which area this bug is related to?

/area registry

## Bug Summary
**Describe the bug:**
When copying the text of a devfile from the devfile registry viewer adds a bunch of whitespace, e.g.:

```
schemaVersion: 2.0.0

metadata:

  name: java-vertx

  version: 1.1.0

  displayName: Vert.x Java

  description: Upstream Vert.x using Java

  icon: https://raw.githubusercontent.com/vertx-web-site/vertx-logo/master/vertx-logo.svg

  tags: ["Java", "Vert.x"]

  projectType: "vertx"

  language: "java"
```

instead of 
```
schemaVersion: 2.0.0
metadata:
  name: java-vertx
  version: 1.1.0
  displayName: Vert.x Java
  description: Upstream Vert.x using Java
  icon: https://raw.githubusercontent.com/vertx-web-site/vertx-logo/master/vertx-logo.svg
  tags: ["Java", "Vert.x"]
  projectType: "vertx"
  language: "java"
```

**To Reproduce:**
1. Visit https://registry.devfile.io/viewer/devfiles/Community+nodejs
2. Select the devfile's text and copy it
3. Paste it into an editor
4. Extra whitespace between each line should be visible

## Expected behavior
There is no additional whitespace between the lines

**Comment by schultzp2020** [comment]
This issue should be fixed when the registry viewer design is updated

**Comment by michael-valdron** [comment]
This issue has been resolved by devfile/devfile-web#40.
